nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/api/operation/GetTaskNamesOperation.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2013 Nuxeo SA (http://nuxeo.com/) and others.
+ * (C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and others.
```

Comment:
I missed it during review but in the future please keep the original copyright date: `2013-2018`